techdocs/README_TECHDOCS.md

```diff
@@ -0,0 +1,120 @@
+
+Creating a `nav` section will allow you to completely control how the navigation for your documentation displays in DevHub, including the text that is shown, grouping pages in categories and sub-categories and including links to external content.
+
+We've included some example `mkdocs.yml` files below which demonstrate how to configure the `nav` section. You may also wish to consult the [MkDocs documentation](https://www.mkdocs.org/user-guide/writing-your-docs/#configure-pages-and-navigation) for detailed information about navigation and other aspects of mkdocs.yml.
+
+### `nav` with categories
+
+Below is an example of a `mkdocs.yml` file with a `nav` section. This `nav` section is structure to arrange the files in categories. In this case the categories are `Getting Started` and `Usage`.
+
+```yaml
+site_name: My TechDocs
+site_description: Documentation for my project
+docs_dir: "docs"
+edit_uri: edit/main/docs/
+plugins:
+    - techdocs-core
+    - ezlinks
+markdown_extensions:
+    - markdown_inline_mermaid
+    - md_in_html
+    -   mkpatcher:
+            location: patcher.py
+nav:
+- Getting Started:
+  - Introduction: index.md
+  - Create an account: account.md
+  - Setup environment: setup.md
+- Usage:
+  - Deploy locally: local.md 
+  - Deploy to OpenShift: openshift.md
+```
+
+### Flat `nav`
+
+Below is another example of a `mkdocs.yml` file. It does not use categories to arrange the documentation pages.
+
+```yaml
+site_name: My Techdocs
+site_description: Documentation for my project
+docs_dir: "docs"
+plugins:
+    - techdocs-core
+    - ezlinks
+markdown_extensions:
+    - markdown_inline_mermaid
+    - md_in_html
+    -   mkpatcher:
+            location: patcher.py
+nav:
+    - Introduction: index.md
+    - Create an account: account.md
+    - Setup environment: setup.md
+    - Deploy locally: local.md
+    - Deploy to OpenShift: openshift.md
+```
+
+## Notes
+
+Note that MkDocs uses `mkdocs.yml`, not `mkdocs.yaml`, although both appear to work.
+Refer to the [mkdocs user guide](https://www.mkdocs.org/user-guide/configuration/) for more details.
+
+## Previewing documentation locally
+
+You will likely want to preview your documentation on your computer. Follow [these instructions](https://github.com/bcgov/devhub-techdocs-publish/blob/main/docs/index.md#how-to-use-the-docker-image-to-preview-content-locally) to do so.
+
+## Building and publishing your documentation
+
+The [GitHub Action workflow file](.github/workflows/publish.yaml) can validate, build (produce browser-friendly HTML from Markdown) and publish your documentation to DevHub automatically.  Initially, it is set up just to validate and build your documentation. 
```

Comment:
This should be updated

Reply:
updated in newest commit - [f74d835](https://github.com/bcgov/citz-imb-sso-express/pull/70/commits/f74d835b112e0adb23c3461aa43e6929f6802bbc)